src/EnergyPlus/HVACSingleDuctInduc.cc

```diff
@@ -764,7 +795,11 @@ namespace HVACSingleDuctInduc {
 				}
 			} else {
 				CheckZoneSizing( IndUnit( IUNum ).UnitType, IndUnit( IUNum ).Name );
-				MaxTotAirVolFlowDes = max( TermUnitFinalZoneSizing( CurZoneEqNum ).DesCoolVolFlow, TermUnitFinalZoneSizing( CurZoneEqNum ).DesHeatVolFlow );
+				if ( CurTermUnitSizingNum > 0 ) {
```

Comment:
No, CurTermUnitSizingNum is *always* used.  If there is no DesignSpecification:AirTerminal:Sizing object then the flows etc in TermUnitFinalZoneSizing will be the same as before.  (FinalZoneSizing plus any system sizing adjustments).  If there is a DesignSpecification:AirTerminal:Sizing object then they get scaled by that also.